src/services/edusharing/hooks/index.js

```diff
@@ -0,0 +1,27 @@
+const auth = require('@feathersjs/authentication');
+const { disallow } = require('feathers-hooks-common');
+const globalHooks = require('../../../hooks');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['globalHooks' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4791331)